crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -345,6 +348,11 @@ impl Starknet {
         Ok(new_block_hash)
     }
 
+    /// Commits the the diff between latest and pending state.
```

Comment:
I think that this comment should include `state and new_historic` (related to StarknetState struct) not `latest and pending state` (related to Starknet struct)

Reply:
Updated comments in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/492/commits/ecb68d6b626cba549df38b5bfe652d259e055371